oak_functions/loader/src/server.rs

```diff
@@ -714,3 +718,31 @@ pub fn format_bytes(v: &[u8]) -> String {
         .map(|s| s.to_string())
         .unwrap_or_else(|_| format!("{:?}", v))
 }
+
+// The EndPoint of a birectional channel. Sender and Receiver are exposed.
+pub struct EndPoint {
+    pub sender: Sender<AbiMessage>,
+    pub receiver: Receiver<AbiMessage>,
+}
+
+// Create a channel with two symmetrial endpoints.  The AbiMessage sent from one EndPoint are
+// received at the other EndPoint and vice versa by connecting two unidirectional
+// tokio::mpsc channels.
+//
+// In ASCII art:
+//  sender _____  ____ sender
+//              \/
+// receiver ____/\____ receiver
+pub fn channel_create() -> (EndPoint, EndPoint) {
+    let (tx0, rx0) = channel::<AbiMessage>(100);
```

Comment:
Could you make 100 a const somewhere? Also perhaps add a comment.

Reply:
Done.